lib/files.js

```diff
@@ -13,15 +13,18 @@ let fs = require('fs'),
 /** Default (asynchronous) file loading function for assembling IDLs. */
 function createImportHook() {
   let imports = {};
-  return function (fpath, kind, cb) {
-    fpath = path.resolve(fpath);
+  return function ({path: fpath, parentPath}, cb) {
```

Comment:
Can we rename to `importerPath` to be more explicit?

Also curious if you considered adding the importer's _parent directory_'s path directly here.